ospfd/ospfd.c

```diff
@@ -2004,29 +1984,9 @@ struct ospf_nbr_nbma *ospf_nbr_nbma_lookup(struct ospf *ospf,
 struct ospf_nbr_nbma *ospf_nbr_nbma_lookup_next(struct ospf *ospf,
```

Comment:
so ... looks like this api always just returns NULL now?

Reply:
well I would say it's returned NULL since 2002.  I go back through and just clean this up a bit more in a sec here.